src/beast/beast/module/core/native/linux_Files.cpp

```diff
@@ -145,9 +144,9 @@ File File::getSpecialLocation (const SpecialLocationType type)
 class DirectoryIterator::NativeIterator::Pimpl
 {
 public:
-    Pimpl (const File& directory, const String& wildCard_)
+    Pimpl (const File& directory, const String& _wildCard)
```

Comment:
I don't care to get in an argument with you here, @vinniefalco , but when I look in the CodingStyle.md on develop (https://github.com/ripple/rippled/blob/develop/doc/CodingStyle.md) I see no such rule.  As a matter of fact this is the first time I've heard the rule you're suggesting.  I'm guessing you need to do a pull request for the style guide?